### ISSUE TITLE
Predict SQL is used inconsistently under xgboost and tf model

### ISSUE THREAD

**Issue by Yancey0623** [sql-machine-learning/sqlflow#741]
For xgboost: `SELECT * FROM xx PREDICT table_name.class`
For tensorflow: `SELECT * FROM xxx PREDICT table_name`

Maybe we need a consistent name after `PREDICT` keyword.

**Comment by tonyyang-svail** [comment]
Agree. I vote for `PREDICT table_name`, which is shorter. The prediction result column name can be the same as the label column name from training.

We will make this consistent during the implementation of IR.

**Comment by typhoonzero** [comment]
This issue is already fixed in the latest version.
